db/migrate/20240712120000_update_group_structure.rb

```diff
@@ -72,7 +72,13 @@ def up
     end
 
     say_with_time("apply the effects of some Role#after_destroy callbacks which might have been circumvented (may take up to an hour!)") do
-      Person.find_each do |person|
+      connection = ActiveRecord::Base.connection
+      sql = <<-SQL
+        SELECT * FROM people
+      SQL
+      people = connection.execute(sql)
+
+      for person in people do
```

Comment:
Das bringt eigentlich wenig, wenn nachher weiter mit Rails-Methoden gearbeitet wird. Weiterhin ist `.find_each` schon eine gute Performanceoptimierung.

Reply:
Wir können aber das `.find_each` nicht verwenden, wenn wir das Model laden bekommen wir einen Fehler, da wir eine ENUM-Column im Model definieren, die erste bei einer späteren Migration in der Datenbank hinzugefügt wird.